src/utils/domUtils.ts

```diff
@@ -39,11 +39,7 @@ export module _Util {
     }
 
     export function isSelectionRemovedFromSVG(selection: D3.Selection) {
-      var n = (<Node> selection.node());
-      while (n !== null && n.nodeName.toLowerCase() !== "svg") {
-        n = n.parentNode;
-      }
-      return (n == null);
+      return selection !== getBoundingSVG(selection);
```

Comment:
Wouldn't actually do the right thing, due to problems in `getBoundingSVG()`; see below.